tests/config/ray-service-serve-update.yaml.template

```diff
@@ -44,30 +44,18 @@ spec:
     headGroupSpec:
       # Kubernetes Service Type, valid values are 'ClusterIP', 'NodePort' and 'LoadBalancer'
       serviceType: ClusterIP
-      # the pod replicas in this group typed head (assuming there could be more than 1 in the future)
```

Comment:
This file should be deleted.

Reply:
https://github.com/ray-project/kuberay/pull/759/files#diff-db7eeddbec6895cf05cb70312b97d42f54bdd61ac34ee18d3f74d02ab349b5bfL274

Reply:
Will do!